src/html/options.html

```diff
@@ -54,6 +54,10 @@
 	<input id="OpenTabsAtEndOfTabBar" type="checkbox"> Open tabs at end of tab bar
 </label>
 
+<label title="Always copy links to clipboard">
+	<input id="CopyLinksToClipboard" type="checkbox"> Copy links instead of opening tabs
+</label>
+
```

Comment:
Replace this by 2 drop-down lists, first for default action, second for modifier action

Reply:
With only one drop-down list when a third option will be implemented how to decide what CTRL will do?

Reply:
Fair point